.github/workflows/contrib-tests.yml

```diff
@@ -17,6 +17,7 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v2
       - name: Run Contrib Tests
+        continue-on-error: true
```

Comment:
It is only the last step/job in the workflow so the continue-on-error has no effect. The PR will still be marked as error I think.

Reply:
Actually the contrib-tests seems green now, will be hard to know when things are broken there.

Reply:
Also contrib-tests is not "required" as you can see we merge PRs with that failing

Reply:
Yea, this is useless.